frontend/src/metabase/entities/data-apps/utils.ts

```diff
@@ -1,5 +1,9 @@
-import { DataApp } from "metabase-types/api";
+import { Collection, DataApp } from "metabase-types/api";
```

Comment:
I really like the `import type` convention for the sake of readability when were are simply importing types as it makes it very obvious that we're only importing these as types. I think there may also be some performance/bundle size improvements because these statements get [fully purged](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export) on build. 

What do you think?

```suggestion
import type { Collection, DataApp } from "metabase-types/api";
```

Reply:
Sounds good to me. Thanks for the suggestion 👍